src/svc-nifid.yml

```diff
@@ -32,6 +34,8 @@ components:
       $ref: './svc/nifid/schemas/Subscriptions.yml'
     SubscriptionStatus:
       $ref: './svc/nifid/schemas/SubscriptionStatus.yml'
+    SubscriptionRunStatus:
```

Comment:
```
 SubscriptionStatusArray:
 $ref: './svc/nifid/schemas/SubscriptionStatusArray.yml'
```

 should be under these schemas as well

Reply:
yes, good catch! I was.. testing, reverting, blah, blah... repeat.. did that and reverted it(apparently).

Thank you!